integration_tests/src/main/python/hybrid_parquet_test.py

```diff
@@ -278,3 +278,41 @@ def test_hybrid_parquet_filter_pushdown_timestamp(spark_tmp_path):
     assert_gpu_and_cpu_are_equal_collect(
         lambda spark: spark.read.parquet(data_path).filter(f.col("a") > f.lit(datetime(2024, 1, 1, tzinfo=timezone.utc))),
         conf=filter_split_conf)
+
+@ignore_order(local=True)
+@pytest.mark.skipif(is_databricks_runtime(), reason="Hybrid feature does not support Databricks currently")
+@pytest.mark.skipif(not is_hybrid_backend_loaded(), reason="HybridScan specialized tests")
+@hybrid_test
+def test_hybrid_parquet_filter_pushdown_aqe(spark_tmp_path):
+    stream_data_path = spark_tmp_path + '/PARQUET_DATA/stream_data'
+    build_data_path = spark_tmp_path + '/PARQUET_DATA/build_data'
+    data_gen_schema = [('key', LongGen(nullable=False, min_val=0, max_val=19)), ('value', long_gen)]
+    with_cpu_session(
+        lambda spark: gen_df(spark, data_gen_schema, length=4096).write.parquet(stream_data_path),
+        conf=rebase_write_corrected_conf)
+    with_cpu_session(
+        lambda spark: gen_df(spark, data_gen_schema, length=512).write.parquet(build_data_path),
+        conf=rebase_write_corrected_conf)
+    conf = filter_split_conf.copy()
+    conf.update({
+        'spark.sql.adaptive.enabled': 'true',
+        'spark.rapids.sql.parquet.pushDownFiltersToHybrid': 'GPU'
```

Comment:
Should be 'spark.rapids.sql.hybrid.parquet.filterPushDown'. This config name in upper cases are also wrong but passed the tests luckily.